pilot/lib/pilot/tester_manager.rb

```diff
@@ -67,7 +67,14 @@ def remove_tester(options)
         # If no groups are passed to options, remove the tester from the app-level,
         # otherwise remove the tester from the groups specified.
         if config[:groups].nil? && tester.kind_of?(Spaceship::Tunes::Tester::External)
-          test_flight_tester = Spaceship::TestFlight::Tester.find(app_id: app.apple_id, email: tester.email)
+          test_flight_testers = Spaceship::TestFlight::Tester.search(app_id: app.apple_id, text: tester.email)
```

Comment:
Is there any chance that a field other than email might match on an email address?  I'm guessing no, since the structure is pretty simple, but it would kind of suck if we removed the wrong tester because of a false-positive.  (one possible work around could be just to filter the found list of testers on found_tester.email = tester.email or something?) . I might be making something out of nothing here, though.